apps/web/src/components/hero/index.tsx

```diff
@@ -27,6 +27,22 @@ function Hero(): React.JSX.Element {
       return
     }
 
+    const dataInStorage: string | null = localStorage.getItem('waitListData')
+    const waitListedEmails: string[] = dataInStorage ? (JSON.parse(dataInStorage) as string[]) : []
+  
+    // actual logic where we are checking if this email is already in waitlisted users or not
+    if (waitListedEmails.includes(email)) {
+      toast.custom(() => (
+        <div className="text-brandBlue border-brandBlue/20 w-[90vw] rounded-lg border bg-[#852b2c] p-2 shadow-2xl backdrop-blur-3xl md:w-[20vw]">
```

Comment:
Not an expert but do we not have any theme color for `bg-[#852b2c]`

Reply:
I have used this color from the toast message which is already been used to show errors. That's why went with this bg. Rest of the changes in the variable names have been done successfully and pushed to the same branch.

Reply:
@poswalsameer will it be possible for you or add this hex code to the tailwind config so we can have uniformity

Reply:
> @poswalsameer will it be possible for you or add this hex code to the tailwind config so we can have uniformity

Sure, that won't be an issue for me. What name should I go with? I was thinking of naming it "errorRed", please let me know about this.

Reply:
You can name it anything relevant

Reply:
> You can name it anything relevant

Hi, done with the change, added a variable for errorRed in tailwind.config.js. You can see the latest commit.